kobolds/biology.md

```diff
@@ -1,58 +1,66 @@
+sharper than the cutting teeth of mammalian sentients) and crushing teeth at the
+rear of their mouths. They have a wide, muscly tongue with a fork near the end
+that is typically black.
 
 They have two arms, two legs and a long, thick tail. They are typically slender
-and, while warm-blooded, have very low body fat percentage with their tail
-accounting for 30-45% of their body mass.
+and have a very low body fat percentage with their tail accounting for 30-45% of
+their body mass. They are mesothermic; They generate heat internally from
+metabolic processes, but have much weaker heat regulation than most mammals.
+They do not, for instance, shiver as a reaction to cold.
 
 They have hands with four fingers and an opposable thumb. They are digitigrades,
 walking on the digits of their legs with long carpals and tarsals keeping their
 ankles far from the ground like canines. Also like canines, they typically only
-have four toes on their legs and pads where they touch the ground. Both fingers
-and toes are clawed, though most individuals keep their finger claws trimmed
-neatly for aesthetic and utilitarian reasons.
+have four toes and pads where they touch the ground. Both fingers and toes are
+clawed, though most individuals keep their finger claws trimmed neatly for
+aesthetic and utilitarian reasons.
 
-Kobolds' reproductive organs are between their legs, below their bellies and in
-front of the anus (which is near underside of the base of their tails). Most of
-the time, all individuals' reproductive organs will look like a long slit in the
-skin. During arousal, this slit expands and opens revealing, typically, either
-labia (note this is not a cloaca, but is much closer to common analogous
-mammalian organs) or a penis (though of course genetic variation accounts for
-less common arrangements). See below for more about kobold sex and reproduction.
-
-Their skin is covered in scales. Most commonly, these are smooth and shiny like
-a gila monster, though rougher on some individuals, especially as they age.
-There is a high variability in appearance beyond this. As with all other
-sentients, physical traits are selected for due to societal concerns and vary
-across the total population of the species. Only the most common will be
-discussed here.
+
+### Scales & Coloration
+
+Their skin is covered in scales. Most commonly, these are smooth and shiny,
+though rougher on some individuals, especially as they age. There is a high
+variability in appearance beyond this. As with all other sentients, physical
+traits are selected for due to societal concerns and vary across the total
+population of the species. Only the most common will be discussed here.
 
 Most kobolds have smooth scales over their entire bodies. Coloration can vary
 widely and is one of the major identifying marks (like skin and hair color among
 mammalian Big People). It is also clearly inherited from parents. For example,
```

Comment:
I think this is the first use of the term "Big People" and I don't know about just suddenly introducing it like this. For some reason when I was originally writing, I switched from "sentients" to "Big People". And I like the kobold idea of "Big People", but I'm not sure if I should use it outside of a discussion of it. Whatever I do, I should probably be more consistent with it one way or the other.